src/modules/itemsCounter.test.js

```diff
@@ -0,0 +1,19 @@
+describe('Total movie items on the DOM', () => {
+  test('Display the total on the header section', () => {
+    document.body.innerHTML = `
+      <header>
+        <span class="totality"></span>
+      </header>
+      <div class="anime-holder"></div>
+      <div class="anime-holder"></div>
+      <div class="anime-holder"></div> 
+        `;
+
+    const movies = document.querySelectorAll('.anime-holder');
+    const allMovies = movies.length;
+    const totalHolder = document.querySelector('.totality');
+    totalHolder.innerHTML = allMovies;
+
+    expect(totalHolder.innerHTML).toEqual('3');
+  });
+});
```

Comment:
- [ ] Your 'itemsCounter' test file is currently not importing the function you created for counting the number of movies/animes on your Home page. Kindly import the function and use it in your test file. Currently, you are only calculating the number of movies with the `allMovies` const, which is not testing any function being used in your application. Please, remember to test edge cases, like if there is no movie/series on your page for example.